api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -98,10 +99,16 @@ public void sendWelcomeEmail(final String contactEmail, final String password, f
   @Override
   public void sendInstitutionUserInstructions(String contactEmail, String userInstructions)
       throws MessagingException {
+    // TODO(RW-6482): Use a templating system rather than manual oneoff escaping.
+    // These institutional instructions are stored unescaped. Though they are input by admins,
+    // the strings should note be trusted as HTML.
```

Comment:
Going to try again with short version of my earlier comment, since that one was erased and I don't feel like rewriting it. The bug you filed is a good discussion point, but doesn't need a TODO here since this code is fine as-is. The rest of the comment is calling out something that should be the norm. Not escaping text demands a comment as to why it is safe to do so. I think this entire comment should be removed.

Reply:
If you view the entirety of this file, you'll notice that this code has special handling in comparison to the other email string substitutions. This is inconsistent with the other lines, and therefore is deserving of a comment in my view so that the next person doesn't come along and remove it for consistency, or have to lookup the code history to understand why it's different. As you noted, these should all be treated uniformly, and the escaping should happen automatically (we don't have legitimate cases where we are injecting arbitrary HTML). I suspect the remaining comment lines can also be deleted once the TODO is resolved, but only because you'd be deleting the special-cased code below.